.github/workflows/build-and-test.yml

```diff
@@ -53,22 +53,22 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest]
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [ 3.6.8, 3.7.9, 3.8.8 ]
```

Comment:
Do we have to pin the exact Python version here? I'm assuming it's necessary for the URL in the curl command below?

Reply:
If I’m not mistaken, the 10.9 python distribution is the official release for all macos versions >= 10.9. So I was thinking, that we just could use that. If you want to limit to 10.14, we need to compile Python for the target.

Regarding the python version for the curl command, I don’t think omitting the patch version would work.